modules/custom/az_paragraphs/az_paragraphs_text_media/js/az_paragraphs_az_text_media_youtube.es6.js

```diff
@@ -0,0 +1,172 @@
+(($, Drupal) => {
+  Drupal.behaviors.az_youtube_video_bg = {
+    attach(context, settings) {
+      if (window.screen && window.screen.width > 768) {
+        // @see https://developers.google.com/youtube/iframe_api_reference
+        // defaults
+        const defaults = {
+          ratio: 16 / 9,
+          videoId: "",
+          mute: true,
+          repeat: true,
+          width: $(window).width(),
+          playButtonClass: "az-video-play",
+          pauseButtonClass: "az-video-pause",
+          muteButtonClass: "az-video-mute",
+          volumeUpClass: "az-video-volume-up",
+          volumeDownClass: "az-video-volume-down",
+          increaseVolumeBy: 10,
+          start: 0,
+          minimumSupportedWidth: 600
+        };
+        const { bgVideos } = settings.azFieldsMedia;
+        const BgVideoParagraphs = document.getElementsByClassName(
```

Comment:
Nitpick: is this variable name capitalized intentionally?  I think capitalized names are generally used for objects / built-ins...

Reply:
testing locally